src/automata/FireAndForgetRequester.h

```diff
@@ -0,0 +1,47 @@
+// Copyright 2004-present Facebook. All Rights Reserved.
+
+#pragma once
+
+#include <memory>
+
+#include "src/Frame.h"
+#include "src/Payload.h"
+#include <src/ConnectionAutomaton.h>
+#include "src/ReactiveStreamsCompat.h"
+
+
+namespace reactivesocket {
+
+/// Implementation of an automaton that represents a Fire-and-Forget requester.
+/// Note that it's simpler than other automata due to the inherent throwaway
+/// simplicity of Fire-and-Forget semantics. Stream IDs are not tracked.
+///
+/// TODO: we need to add logging similar to other *Requester automata.
+class FireAndForgetRequester : public Subscription {
```

Comment:
Does this need to be a Subscription? Can this type just have onError and onComplete?